scripts/cat/history.py

```diff
@@ -443,6 +439,8 @@ def add_murder(self, murderer_id, victim):
             self.murder["is_murderer"] = []
         if "is_victim" not in victim.history.murder:
             victim.history.murder["is_victim"] = []
+        else:
+            print(f"WARNING: victim cat: {victim.ID} has no victim history.")
```

Comment:
```suggestion
            print(f"WARNING: victim cat: {victim.ID} already has a murder history - as the victim!")
```
I do also think there's a possible case for an intentional crash here rather than just a debug print. but eh.